smqtk_detection/impls/detect_image_objects/mmdet_base.py

```diff
@@ -0,0 +1,275 @@
+import logging
```

Comment:
This file probably doesn't need to be called "mmdet_*base*.py": we can drop the "base" part since it doesn't seem that we need to subclass this like the detectron2 impl needed.